source/MaterialXCore/Element.h

```diff
@@ -71,6 +71,92 @@ using ElementMap = std::unordered_map<string, ElementPtr>;
 /// A standard function taking an ElementPtr and returning a boolean.
 using ElementPredicate = std::function<bool(ConstElementPtr)>;
 
+/// @class ElemenEquivalenceResult
+/// The results of comparing for equivalence.
+class MX_CORE_API ElementEquivalenceResult
+{
+  public:
+      ElementEquivalenceResult() = default;
```

Comment:
You may want to double-check the formatting of this new class, as I'm seeing some unusual indentation here.